Cpp/odin/odin.cpp

```diff
@@ -61,6 +62,9 @@ const fostlib::setting<fostlib::string> odin::c_facebook_endpoint(
 const fostlib::setting<fostlib::json> odin::c_google_aud(
         "odin/odin.cpp", "odin", "Google", fostlib::null, true);
 
+const fostlib::setting<fostlib::string> odin::c_app_namespace(
+        "odin/odin.cpp", "odin", "Application namespace", "https://odin.hot-now.com/app/", true);
```

Comment:
This needs to be a generic name here, we have to set the real one in our configuration.